webapp/public/asseteditor.html

```diff
@@ -32,8 +29,58 @@
     <script type="text/javascript" src="/blb/pxtapp.js"></script>
     <script>
         pxt.setAppTarget(pxtTargetBundle);
-        // TO DO: get palette info from message from vs code extension
+        // This can be overridden via the messages sent to the page
         pxt.appTarget.runtime.palette = ["#000000","#ffffff","#ff2121","#ff93c4","#ff8135","#fff609","#249ca3","#78dc52","#003fad","#87f2ff","#8e2ec4","#a4839f","#5c406c","#e5cdc4","#91463d","#000000"];
+
+        // this get rewritten to blob URLs with SHAs while uploading to the cloud
+        // keep in sync with index.html
+        window.MonacoPaths = {
+            "vs/loader": "/blb/vs/loader.js",
+            "vs/base/worker/workerMain": "/blb/vs/base/worker/workerMain.js",
+            "vs/basic-languages/bat/bat": "/blb/vs/basic-languages/bat/bat.js",
+            "vs/basic-languages/cpp/cpp": "/blb/vs/basic-languages/cpp/cpp.js",
```

Comment:
we don't, i just copied this directly from the index.html because it's easier to keep them in sync than to cherry pick the ones we need. adding it here won't cause it to be loaded or anything